src/client/components/board/board.js

```diff
@@ -107,6 +107,14 @@ class Board extends React.Component {
           </div>
         </div>
         <Sidebar playerID={this.props.playerID} gameID={this.props.gameID} G={this.props.G} ctx={this.props.ctx} moves={this.props.moves} phase={this.props.ctx.phase} current={current} active={active} names={this.state.names} />
+        <Threatbar playerID={this.props.playerID} model={this.state.model} names={this.state.names} G={this.props.G} ctx={this.props.ctx} moves={this.props.moves} active={active} />
+        <p className="license">The card game
```

Comment:
Maybe refactor this `<p />` into its own component? Then it might be easier to display a different text depending on whether the gameMode is EoP or Cornucopia.